src/lib/pages/backoffice/Loan/LoanDetails/LoanOverbooked/LoanOverbooked.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { Segment } from 'semantic-ui-react';
+import { documentApi } from '@api/documents';
+
+export default class LoanOverbooked extends Component {
```

Comment:
I would propose to be more specific:
```suggestion
export default class LoanOverbookedWarning extends Component {
```